webapp-django/crashstats/api/views.py

```diff
@@ -415,9 +409,9 @@ def _describe_model(model_name, model):
     params = list(model_inst.get_annotated_params())
     params.sort(key=lambda x: (not x["required"], x["name"]))
     methods = []
-    if model.get:
+    if hasattr(model, "get"):
         methods.append("GET")
-    elif model.post:
+    elif hasattr(model, "post"):
         methods.append("POST")
```

Comment:
This should be `getattr(model, "get", None)` because there are some models where `get = None`.